utils/minify/Gruntfile-rtl.js

```diff
@@ -0,0 +1,57 @@
+module.exports = function(grunt)
```

Comment:
Can't we just have one Gruntfile to generate both the minified files to avoid repetitions ? Looks like the `concat.dist.files` and `uglify.dist.files` options take are dictionaries that would allow specifying both the output files in the same Gruntfile.

Reply:
Any news on this ?